modules/ROOT/pages/tokenization-formats.adoc

```diff
@@ -114,6 +117,9 @@ a| * Decimal
 |*Preserve serial number sub-component* 
 |This option is for the social security data domain only.
 |Social security number
+
+|*Masking character*
+|This option
```

Comment:
Was this sentence cut off? All I see is "This option"

Reply:
Yes, I needed the info on the Data Domain which you provided in Slack. I'll update now.